Tools/CISupport/ews-build/events.py

```diff
@@ -313,14 +313,19 @@ def stopService(self):
 
 class GitHubEventHandlerNoEdits(GitHubEventHandler):
     ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
+    OPEN_STATES = ('open',)
 
     def _get_commit_msg(self, repo, sha):
         return ''
 
     def handle_pull_request(self, payload, event):
         pr_number = payload['number']
         action = payload.get('action')
+        state = payload.get('state')
         if action not in self.ACTIONS_TO_TRIGGER_EWS:
             log.msg('Action {} on PR #{} does not indicate code has been changed'.format(action, pr_number))
             return ([], 'git')
+        if state not in self.OPEN_STATES:
+            log.msg("PR #{} is '{}', which triggers nothing".format(action, state))
```

Comment:
Do you want to print a pr number here (after PR #)?

Reply:
Yes, should be `pr_number` instead of `action`